controller-client/src/main/java/org/jboss/as/controller/client/RemotingModelControllerClientPermission.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2017 JBoss by Red Hat.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.as.controller.client;
+
+import java.security.BasicPermission;
+
+/**
+ * Permission to execute remote management calls on a server.
+ * <p>
+ * A permission contains a name (also referred to as a "target name") but
+ * no actions list; you either have the named permission or you don't.
+ * </p>
+ *
+ * <p>
+ * The target name is the name of the permission. The following table lists all the possible {@link org.jboss.as.controller.client.RemotingModelControllerClientPermission} target names,
+ * and for each provides a description of what the permission allows.
+ * <ul>
+ * <li>performRemoteCall: permission to perform a remote call on the server model controller</li>
+ * </ul>
+ * </p>
+ * @author Emmanuel Hugonnet (c) 2017 Red Hat, inc.
+ */
+public class RemotingModelControllerClientPermission extends BasicPermission {
```

Comment:
s/Remoting/Remote

"Remoting" is an implemenation detail; ie part the name of the lib we use to allow remote management.